commerce_coordinator/apps/core/constants.py

```diff
@@ -8,6 +8,28 @@ class Status:
     UNAVAILABLE = "UNAVAILABLE"
 
 
+class OrderState(Enum):
+    """
+    Enum for order states.
+    """
+    CART = 'cart'
+    ADDRESS = 'address'
+    PAYMENT = 'payment'
+    COMPLETE = 'complete'
+    CANCELED = 'canceled'
+
+
+class OrderPaymentState(Enum):
```

Comment:
- [ ] Add ticket to have frontend-app-payment look for OrderPaymentState instead of PaymentState when polling.

Reply:
Created [THES-267](https://2u-internal.atlassian.net/browse/THES-267).